packages/apsara-ui/src/InfiniteScroll/InfiniteScroll.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useRef } from "react";
+
+interface InfiniteScrollProps<T> {
+    renderItem: (item: T) => React.ReactNode;
+    onBottomScroll: () => void;
+    items: T[];
+    height?: string;
+    style?: React.CSSProperties;
+    isLoading?: boolean;
+    noMoreData?: boolean;
+    containerRef?: React.RefObject<HTMLElement>;
+    threshold?: number;
+    loadingComponent?: React.ReactNode;
+    noMoreDataComponent?: React.ReactNode;
+}
+
+const InfiniteScroll = <T,>({
+    renderItem,
+    onBottomScroll,
+    items,
+    style,
+    height,
+    isLoading,
+    noMoreData,
+    containerRef,
+    threshold = 1,
+    loadingComponent,
+    noMoreDataComponent,
+}: InfiniteScrollProps<T>) => {
+    const defaultContainerRef = useRef<HTMLDivElement>(null);
+    const containerElem = containerRef ? containerRef.current : defaultContainerRef.current;
+
+    useEffect(() => {
+        if (!containerElem) return;
+
+        const onScroll = () => {
+            if (isBottom(containerElem, threshold)) {
+                onBottomScroll();
+            }
+        };
+
+        containerElem.addEventListener("scroll", onScroll);
+
+        return () => {
+            containerElem.removeEventListener("scroll", onScroll);
+        };
+    }, [containerElem, onBottomScroll, threshold]);
+
+    const loadingComp = loadingComponent || <DefaultLoading />;
+
+    const scrollStyle: React.CSSProperties = !containerRef
+        ? {
+              height: height || "100%",
```

Comment:
```suggestion
              height: height || "calc(100% - 0)",
```

we cannot scroll `height : 100%`;